internal/pkg/manifest/worker_svc.go

```diff
@@ -86,6 +87,18 @@ func (q *SQSQueueOrBool) IsEmpty() bool {
 	return q.Advanced.IsEmpty() && q.Enabled == nil
 }
 
+// UnmarshalYAML implements the yaml(v3) interface. Here it sets default value
+// of the type field if it is not set already.
+func (t *Topic) UnmarshalYAML(value *yaml.Node) error {
```

Comment:
Just to mention that we might want to be careful with setting default values in `UnmarshalYAML` in the future because it could easily mess up environment override, which takes advantage of `nil` pointers to determine if a value has override.

In this case we should be fine since `Topic` struct isn't stand-alone and is instead always in a list of `Topic`s - i.e. there is no `Topic`-level environment override.

Reply:
That's a great call out.